README.md

```diff
@@ -1,162 +1,10 @@
-params.
-
-	# running sha-perf with default params
-	boot and execute on your target
-	$ tee-supplicant &
-	$ xtest --sha-perf
-
-	# getting usage details and list of possible options for sha-perf
-	$ xtest --sha-perf -h
-
-#### Compiler flags
-To be able to see the full command when building you could build using following
-flag:
-
-`$ make V=1`
-
-To state where build files are stored use the `O` flag.
-
-`$ make O=$HOME/foo`
-
-By default `optee_test` expects that `optee_client` is located at the same
-folder level. However if you build optee_client in another location, then you
-also would need to use (or export) the following flag:
-
-`$ make OPTEE_CLIENT_EXPORT=$HOME/my_new_location/out/export`
-
-## Coding standards
-In this project we are trying to adhere to the same coding convention as used in
-the Linux kernel (see
-[CodingStyle](https://www.kernel.org/doc/Documentation/CodingStyle)). We achieve this by running
-[checkpatch](http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/scripts/checkpatch.pl) from Linux kernel.
-However there are a few exceptions that we had to make since the code also
-follows GlobalPlatform standards. The exceptions are as follows:
-
-- CamelCase for GlobalPlatform types are allowed.
-- And we also exclude checking third party code that we might use in this
-  project, such as LibTomCrypt, MPA, newlib (not in this particular git, but
-  those are also part of the complete TEE solution). The reason for excluding
-  and not fixing third party code is because we would probably deviate too much
-  from upstream and therefore it would be hard to rebase against those projects
-  later on (and we don't expect that it is easy to convince other software
-  projects to change coding style).
-
-### checkpatch
-Since checkpatch is licensed under the terms of GNU GPL License Version 2, we
-cannot include this script directly into this project. Therefore we have
-written the Makefile so you need to explicitly point to the script by exporting
-an environment variable, namely CHECKPATCH. So, suppose that the source code for
-the Linux kernel is at `$HOME/devel/linux`, then you have to export like follows:
-
-`$ export CHECKPATCH=$HOME/devel/linux/scripts/checkpatch.pl`
-
-You also have to give the path to the `optee_os` repository:
-
-`$ export OPTEE_OS_SRC=/path/to/optee_os`
-
-In case `OPTEE_OS_SRC` is unset or empty, the dispatcher script will default to `../optee_os`.
-
-Thereafter it should be possible to run checkpatch manually from the repository root as:
-
-`$ ./scripts/checkpatch.sh -h`
-
-There are also targets for common use cases in the [Makefile](Makefile):
-
-```
-make checkpatch			#check staging and working area
-make checkpatch-staging #check staging area (added, but not committed files)
-make checkpatch-working #check working area (modified, but not added files)
-```
\ No newline at end of file
+[optee_test]: https://optee.readthedocs.io/building/gits/optee_test.html
```

Comment:
Unfortunately, this URL doesn't work, you have to keep the `en/latest/` part: `https://optee.readthedocs.io/en/latest/building/gits/optee_test.html`

Reply:
Took the URL from master, fixed in https://github.com/OP-TEE/optee_test/pull/393, will cherry-pick onto this PR.